src/lib.rs

```diff
@@ -1,18 +1,23 @@
 extern crate rustc_serialize;
+use std::string;
 
 pub enum Value {
   Boolean(bool),
   TinyInt(u8),
+  TinyText(Result<String, string::FromUtf8Error>)
```

Comment:
Sorry, child breakdown.  Also we should probably be turning the `FromUtf8Error` into that `PackStreamError`

I think... ;)